core/src/mindustry/entities/abilities/EnergyFieldAbility.java

```diff
@@ -28,7 +28,7 @@ public class EnergyFieldAbility extends Ability{
     public float x, y;
     public boolean targetGround = true, targetAir = true, hitBuildings = true, hitUnits = true;
     public int maxTargets = 25;
-    public float healPercent = 3f;
+    public float healPercent = 1f;
```

Comment:
This is not the right way to do this, you are changing the default setting, which will affect mods. Change it in UnitTypes::aegires instead. (add a line inside the init block `healPercent = 1f`)

Reply:
Tysm! (For others reading this has been fixed)